docs/admin/deployment.md

```diff
@@ -562,29 +634,66 @@ PCI:0000:da:00.0 Model:INTEL SSDPED1K750GA  FW:E2010435 Socket:1 Capacity:750 GB
     Device Reliability: OK
     Read Only: OK
     Volatile Memory Backup: OK
+  Intel Vendor SMART Attributes:
+    Program Fail Count:
+       Normalized(%%):100
+       Raw:0
+    Erase Fail Count:
+       Normalized(%%):100
+       Raw:0
+    Wear Leveling Count:
+       Normalized(%%):100
+       Min:0
+       Max:9
+       Avg:3
+    End-to-End Error Detection Count:0
+    CRC Error Count:0
+    Timed Workload, Media Wear(%%):63
+    Timed Workload, Host Reads:65535
+    Timed Workload, Timer:65535
+    Thermal Throttle Status(%%):0
+    Thermal Throttle Event Count:0
+    Retry Buffer Overflow Counter:0
+    PLL Lock Loss Count:0
+    NAND Bytes Written:222897
+    Host Bytes Written:71
+
 ```
 
-The next step consists of adjusting in the server configuration the storage
-devices that should be used by DAOS. The `servers` section of the yaml is a
-list specifying details for each DAOS I/O instance to be started on the host
-(currently a maximum of 2 per host is imposed).
+#### Selection
+
+The next step consists of specifying the devices that should be used by DAOS
+in the server configuration YAML file.
+The `engines` section of the config is a list with details for each DAOS engine
+to be started on the host (currently a maximum of 2 per host is imposed).
```

Comment:
There hasn't been a limit on the number of storage engines for quite a while. Probably a good idea to fix that now.